docs/source/user_guide/index.md

```diff
@@ -22,3 +22,15 @@ maxdepth: 2
 ---
 fundamentals/index
 ```
+
+```{toctree}
+---
+caption: Example notebooks
+maxdepth: 1
+---
+../examples/notebooks/getting_started/index
+../examples/notebooks/creating_models/index
```

Comment:
Should we also display thumbnails directly in the user guide page?

Reply:
Good idea, done